lukasz-lapczyk/review-dealer
# Add i18n

# Overview 
Internationalization was added to site. Two languages are available - english as default and polish. 

# Testing prerequisites
bundle install

# Testing
- navigate to root directory
- log in with 'user_1@example.org', pass: Secret19 or create new account
- choose 'PL' from languages list on navbar
- [x] site should be displayed in Polish
